src/release/main.d

```diff
@@ -48,6 +48,16 @@ void main ( string[] params )
 
     checkOAuthSetup();
 
+    try
+    {
+        import release.gitHelper;
+        getRemote(getUpstream());
+    }
+    catch (Exception exc)
+    {
+        writefln("Warning: %s", exc.msg);
```

Comment:
Shouldn't it exit? I mean how can it possibly succeed further if remote is not valid?